packages/noya-state/src/selectors/__tests__/__snapshots__/gradientSelectors.test.ts.snap

```diff
@@ -0,0 +1,65 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`get Gradient Point angular gradient 1`] = `
+Array [
+  Object {
+    "color": Object {
+      "_class": "color",
+      "alpha": 1,
+      "blue": 1,
+      "green": 1,
+      "red": 1,
+    },
+    "point": Object {
+      "x": 100,
+      "y": 50,
+    },
+  },
+  Object {
```

Comment:
We should add another point in the middle of our gradient, since `from`/`to` are sort of special cases that may not use `lerp`